Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM mambaorg/micromamba:0.19.1
+
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+ENV PATH /usr/local/nvidia/bin/:$PATH
+ENV LD_LIBRARY_PATH /usr/local/nvidia/lib:/usr/local/nvidia/lib64
+
+# Tell nvidia-docker the driver spec that we need as well as to
+# use all available devices, which are mounted at /usr/local/nvidia.
+# The LABEL supports an older version of nvidia-docker, the env
+# variables a newer one.
+ENV NVIDIA_VISIBLE_DEVICES all
+ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
+LABEL com.nvidia.volumes.needed="nvidia_driver"
+
+WORKDIR /
+
+# Make this dir to install JDK
+RUN mkdir -p /usr/share/man/man1
+
+# Install base packages.
+RUN apt-get update --fix-missing && apt-get install -y \
```

Comment:
Should be `apt update` and `apt install`. Also, the `--fix-missing` doesn't seem to work with this image so I would delete it.